provisioning_templates/snippet/ssh_authorized_keys.erb

```diff
@@ -0,0 +1,38 @@
+<%#
+kind: snippet
+name: ssh_authorized_keys
+model: ProvisioningTemplate
+snippet: true
+%>
+
```

Comment:
This generates a lot of whitespace. Could you have a look at the various blocks?